docs/source/pythonapi/examples/mdgxs-part-ii.ipynb

```diff
@@ -822,178 +822,27 @@
    },
    "outputs": [
     {
-     "data": {
```

Comment:
So it looks like a lot was removed from the output of this notebook, rather than simply replaced with a newer run. Did you run this notebook to completion? Or just run a portion of the notebook? Or am I missing something?